dbt/README.md

```diff
@@ -44,8 +44,7 @@ use to predict property values.
 
 ### 2. Define integrity checks that specify the correct format for our data
 
-The tests defined in the `schema.yml` files in the `models/` directory,
-alongside some one-off tests defined in the [`tests/`](./tests/) directory,
```

Comment:
We don't actually have any tests in the `tests/` directory anymore, so there's no use documenting them.